packages/block-library/src/html/index.js

```diff
@@ -30,7 +30,7 @@ export const settings = {
 	attributes: {
 		content: {
 			type: 'string',
-			source: 'html',
+			source: 'raw',
```

Comment:
I understand the reasoning, I was wondering why do we need a specific matcher though. Can't we just say if the `source` is `html` and there's no `selector` we just return the `innerHTML`? (could even be pushed as a fix upstream in `hpq`?

I'll defer to @aduth though for more insights.